keras/backend/ops.py

```diff
@@ -0,0 +1,121 @@
+from ..layers import Lambda
+from ..engine.topology import merge
+from .. import backend as K
+
+
+def _slice_shape(shape, slices):
+    '''slice a shape
```

Comment:
Not really clear what this method does given the docstring. Please add a proper docstring, with and `# Arguments` section, and examples to go along anything that isn't obvious.